lib/iris/tests/test_basic_maths.py

```diff
@@ -553,8 +553,9 @@ def test_square_root(self):
 
         e = a ** 0.5
 
-        self.assertCML(e, ("analysis", "sqrt.cml"))
-        self.assertArrayEqual(e.data, a.data ** 0.5)
+        self.assertArrayAlmostEqual(e.data, a.data ** 0.5)
+        # self.assertArrayEqual(e.data, a.data ** 0.5)
+        # self.assertCML(e, ("analysis", "sqrt.cml"))
```

Comment:
Yeah, can't explain this failure when run on CI - it's not repeatable locally.

Reply:
Okay, discovered that this failure was the result of dtype quantization, given the test data being used in the maths operation.

In summary, the `(73, 96)` of `float32` data resulted in `4` values out of `7,008` being inaccurate with a tolerance of approximately `e-07`.

Simply increasing the `dtype` resolves this, hence a different CML data checksum.